channels/pkg/channels/addon.go

```diff
@@ -25,23 +25,50 @@ import (
 	"net/url"
 )
 
+// Addon is a wrapper around a single version of an addon
```

Comment:
Can't say how much I love this enough. Using the type name as the first word, and adding docs.

🎉